### ISSUE TITLE
[BUG] You can open doors without a keycard after equipping it in the loading screen

### ISSUE THREAD

**Issue by Axiewu** [Manufacturing-Department/issue-tracker#84]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Location

Live

### Game Version

919 - Fearless Glacicon 

### Expected Behavior

What's happening? Once you go into loading screen, which is basically picking teams, weapons etc. and wait for your tools to load, equip keycard then spawn, you can open doors without using a keycard. 
Expected Behaviour? You cannot open them or equip items in the loading menu.

### Steps To Reproduce

1. Join pathos, wait for your screen to load
2. Pick a team and stuff
3. Once your tools load, pull out your keycard
4. Spawn and try to open doors without pulling it out.

### Anything else?

![image](https://user-images.githubusercontent.com/99291801/154802277-ca2616e5-62b1-4391-9f33-5644b4808af4.png)
Something similar was found, exactly once you double m with equipped keycard.
![image](https://user-images.githubusercontent.com/99291801/154802295-87102efa-4c46-40a0-b8a5-adbef3850940.png)

**Comment by FakadoDev** [comment]
Yup this is similar to this [https://github.com/Manufacturing-Department/issue-tracker/issues/56](bug) but is achieved in a different way.

**Comment by FakadoDev** [comment]
Can reproduce. Fakado#8525 - UncleRamenNoodles.
